app/services/sso/apache.rb

```diff
@@ -2,11 +2,11 @@ module SSO
   class Apache < Base
     delegate :session, :to => :controller
 
-    CAS_USERNAME = 'REMOTE_USER'
+    CAS_USERNAME = 'HTTP_REMOTE_USER'
```

Comment:
I like this constant a lot more than duplicating it everywhere. Not going to block on it, but I'd like to see similar constants for `HTTP_REMOTE_USER_GROUPS` + email + firstname + lastname.

However, it is weird that the constant is only used here and not in ApplicationController.